packages/better-php-doc-parser/src/Printer/MultilineSpaceFormatPreserver.php

```diff
@@ -54,7 +54,8 @@ public function resolveCurrentPhpDocNodeText(Node $node): ?string
     public function fixMultilineDescriptions(
         AttributeAwareNodeInterface $attributeAwareNode
     ): AttributeAwareNodeInterface {
-        if (! $attributeAwareNode->getAttribute(Attribute::ORIGINAL_CONTENT)) {
+        $attributeAwareNodeContent = $attributeAwareNode->getAttribute(Attribute::ORIGINAL_CONTENT);
```

Comment:
```suggestion
        $attributeAwareNodeOriginalContent = $attributeAwareNode->getAttribute(Attribute::ORIGINAL_CONTENT);
```

The value should be same as constant name `::<...>`

Reply:
implemented.